### DISCUSSION TITLE
Any ETA on Turbopack disk caching?

### DISCUSSION THREAD

**Discussion by meds** [vercel/next.js#70729] [Help]
### Summary

In my use case Webpack is more reliable and compiles faster than Turbopack, from memory a NextJS dev mentioned this was due to disk caching support in Webpack and that Turbopack would be faster once it gets disk caching support.

Was just curious if this was still on the road map?

**Comment by suprunchuk** [comment]
Yes, disk caching support is still on the roadmap for Turbopack. The Next.js team and the developers behind Turbopack are actively working on improving its performance and feature set, including adding disk caching. Disk caching can significantly enhance build speeds by storing intermediate build artifacts on disk, allowing subsequent builds to reuse these artifacts instead of recomputing them from scratch.

While Turbopack is designed to be a fast and efficient bundler, it is still in active development, and features like disk caching are part of the planned improvements to make it even more competitive with established tools like Webpack. Keep an eye on the official Next.js and Turbopack repositories or their release notes for updates on when this feature becomes available.

**Comment by yogasanas** [comment]
Hi any updates on this?
